src/test/java/com/ays/assignment/service/impl/AssignmentServiceImplTest.java

```diff
@@ -402,4 +403,86 @@ void givenValidAssignmentId_whenAssignmentNotExists_thenThrowAysAssignmentNotExi
         Mockito.verify(identity, Mockito.times(1))
                 .getInstitutionId();
     }
+
+    @Test
+    void givenValidUserId_whenUserHasAssignmentWithValidStatus_thenReturnAssignment() {
```

Comment:
Servise herhangi bir user id parametresi verilmediği için methodu `whenUserHasAssignmentWithValidStatus_thenReturnAssignment` olarak isimlendirebiliriz.

Reply:
Değiştirdim